### ISSUE TITLE
Talk Proposal: PWAs with Next.js, configuration is king. Or is it?

### ISSUE THREAD

**Issue by imbhargav5** [berlinjs/berlinjs.org#268]
##### Abstract

Next.js maintains a fine balance between configs and code and that helps us build websites with the important features like SSR, async data fetching and seamless integration with other libraries seamlessly. But with great unconfigurable internal config come great hurdles. Let's discuss the good and bad of the Next.js. And then talk about how we can overcome some of the hurdles or work around them to build great PWAs.

###### Means to get in touch with you

[Email](bhargavponnapalli.5@gmail.com)
[Twitter](https://twitter.com/imbhargav5)

**Comment by rmehner** [comment]
Hey there,

thanks for the proposal. Right now we have a waitlist that keeps us filled until October and maybe even November. I'd ping you if something frees up in November. Sounds super interesting :)

**Comment by rmehner** [comment]
Just realized that your location says that you're located in India. Is this still true or are you in Berlin now? :)

**Comment by imbhargav5** [comment]
Hi Robin @rmehner , Yes I am in India. I am willing to travel to Berlin with/without sponsorship to present there. Sponshorship would be great, but otherwise, I am willing to spend from my pocket to come there. Giving a presentation there and interacting with you all there is important for me personally.

**Comment by carolstran** [comment]
Hey @imbhargav5, do you have any plans to make it out to Berlin in 2019? 😊 Otherwise, I'll put this issue on hold until we know more about availability/potential sponsorship.

**Comment by imbhargav5** [comment]
Yes definitely. 👌

**Comment by carolstran** [comment]
Do you mean that yes you definitely have plans to come to Berlin? Do you know generally what time of the year?

**Comment by imbhargav5** [comment]
No. I haven't planned the trip yet.  Would like to coordinate with you. I intend to come in either March or April.

**Comment by carolstran** [comment]
It looks like April could work well for us actually! March is currently full but I'm sure we could move things around if that's the only time that you're able to come.

**Comment by carolstran** [comment]
Sorry misunderstood a bit before 😅

**Comment by imbhargav5** [comment]
April works for me definitely. 👍

**Comment by carolstran** [comment]
Definitely? Wow cool that was easy! I'll put you down for April, but just let us know if your plans change 😁 Excited to have you!

**Comment by imbhargav5** [comment]
Yes! So I would like change the title and content of my talk over the course of the next few days. Since then, I got the opportunity to join styled-components as a core member and my mind exploded with various talks I could give around this topic. Would you mind?

**Comment by carolstran** [comment]
Of course, that's no problem! Just send us the new title and description when you have a chance. Congrats on joining the styled-components team 🎉

**Comment by carolstran** [comment]
Just changing the label re: our conversation through Twitter. We still have you down for Q4 and will check in again during the summer 😊

**Comment by carolstran** [comment]
👋 @imbhargav5, just marking this as `On Hold` since you're from out of town. But as we mentioned before, just give us a heads up when you know that you'll be in town and we can try to sort something out!
